e2e/cypress/integration/public_access.spec.js

```diff
@@ -14,18 +15,19 @@ describe('Public Access', () => {
 
     after(() => {
         cy.login();
+        cy.wait(2000);
         cy.delete_dataset('test-dataset-1')
         cy.delete_organization('test-organization')
     })
 
     it('Cannot access the standard public pages', () => {
+        cy.logout();
         cy.request({
             url: '/dataset',
             failOnStatusCode: false
         }).then((response) => {
             // TODO: local extension should return 403 on anonymous access
```

Comment:
We can delete this line too!

Reply:
By "this", I mean this line: `// TODO: local extension should return 403 on anonymous access`

Reply:
👍